src/index.ts

```diff
@@ -1,13 +1,86 @@
 import * as ts from "typescript";
-import { PluginImpl } from "rollup";
+import * as path from "path";
+import { PluginImpl, SourceDescription, InputOptions } from "rollup";
 import { Transformer } from "./Transformer";
 import { NamespaceFixer } from "./NamespaceFixer";
 
+const dts = ".d.ts";
+const tsx = /\.tsx?$/;
+
+const formatHost: ts.FormatDiagnosticsHost = {
+  getCurrentDirectory: () => ts.sys.getCurrentDirectory(),
+  getNewLine: () => ts.sys.newLine,
+  getCanonicalFileName: ts.sys.useCaseSensitiveFileNames ? f => f : f => f.toLowerCase(),
+};
+
+const getCompilerOptions = (input: string): ts.CompilerOptions => {
```

Comment:
```suggestion
function getCompilerOptions(input: string): ts.CompilerOptions {
```
my personal preference is to have function declarations on the toplevel